src/App.tsx

```diff
@@ -1,15 +1,23 @@
 import React from 'react';
-import {BrowserRouter as Router, Routes, Route} from 'react-router-dom';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import './App.scss';
 import Home from './Home/Home';
+// line below is just an example of what to import to use the DisplayPictureOfDay function
```

Comment:
Please remove these comments and the the before merging - they might be confusing for some people